src/main/java/com/cburch/logisim/std/arith/ShifterHdlGeneratorFactory.java

```diff
@@ -191,48 +159,41 @@ public LineBuffer getModuleFunctionality(Netlist TheNetlist, AttributeSet attrs)
           .pair("bitsShiftDiff", (nrOfBits - nrOfBitsToShift))
           .pair("bitsShiftDiff1", (nrOfBits - nrOfBitsToShift - 1))
           .pair("nrOfBitsToShift", nrOfBitsToShift)
-          .pair("nrOfBitsToShift1", nrOfBitsToShift - 1);
-
-    contents.add("""
-        -----------------------------------------------------------------------------
-        --- Here stage {{stageNumber}} of the binary shift tree is defined
-        -----------------------------------------------------------------------------
-
-        """);
-
+          .pair("nrOfBitsToShift1", nrOfBitsToShift - 1)
+          .empty()
+          .addRemarkBlock(String.format("Stage %d of the binary shift tree is defined here", stageNumber));
     if (stageNumber == 0) {
       contents
           .add("""
-            s_stage_0_shiftin <= DataA({{nrOfBits1}}) WHEN {{shiftMode}} = 1 OR {{shiftMode}} = 3 ELSE
-                                 DataA(0) WHEN {{shiftMode}} = 4 ELSE '0';
+            s_stage0ShiftIn <= dataA({{nrOfBits1}}) {{when}} {{shiftMode}} = 1 {{or}} {{shiftMode}} = 3 {{else}}
+                               dataA(0) {{when}} {{shiftMode}} = 4 {{else}} '0';
 
-            s_stage_0_result  <= DataA
+            s_stage0Result  <= dataA
             """)
           .add(
               (nrOfBits == 2)
-                  ? "                        WHEN ShiftAmount = '0' ELSE"
-                  : "                        WHEN ShiftAmount(0) = '0' ELSE")
+                  ? "                      {{when}} shiftAmount = '0' {{else}}"
```

Comment:
wouldn't it be more readable to move {{else}} to next line and align with {{when}}? (multiple)

Reply:
This is a coding convention question, I'll leave it as is for the moment until we decide on the VHDL coding convention.